docs/internal_api.yml

```diff
@@ -241,6 +241,45 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /tenants/{tenant_id}/devices/{device_id}/reindex:
+    post:
+      tags:
+        - Internal API
+      summary: Start reindexing device attributes.
+      operationId: Start Re-indexing
+      parameters:
+        - in: query
+          name: service
+          schema:
+            type: string
+            example: "inventory"
```

Comment:
technically we accept only "Devicemonitor" atm, so maybe let's not mislead in the example?

Reply:
fixed